Source/Engines/ConfigFilters.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RealFuels
+{
+    [KSPAddon(KSPAddon.Startup.AllGameScenes, false)]
+    public class ModuleShowInfoUpdater : MonoBehaviour
+    {
+        protected bool run = true;
+        private void Update()
+        {
+            if (run)
+            {
+                Debug.Log("[RealFuelsFilters] Updated Info boxes");
+                foreach (AvailablePart ap in PartLoader.LoadedPartsList)
+                {
+                    if (ap.partPrefab.FindModuleImplementing<ModuleEngineConfigs>() is ModuleEngineConfigs mec)
+                    {
+                        foreach (AvailablePart.ModuleInfo x in ap.moduleInfos)
+                        {
+                            if (x.moduleName.Equals("Engine Configs"))
+                            {
+                                x.info = mec.GetInfo();
+                            }
+                        }
+                    }
+                }
+
+                run = false;
+                GameObject.Destroy(this);
+            }
+        }
+    }
+
+    public class ConfigFilters
+    {
+        public class Filter
+        {
+            public string filterID;
+            public Func<ConfigNode, bool> criteria;
+
+            public Filter(string filterID, Func<ConfigNode, bool> criteria)
+            {
+                this.filterID = filterID;
+                this.criteria = criteria;
+            }
+        }
+
+        public class FilterList
```

Comment:
`Dictionary<string, Func<ConfigNode, bool>>` might be more appropriate?

Reply:
Also, I think it might be more idiomatic to make a named delegate here?

Reply:
I have changed to a Dict, but not to a named Delegate. Would that make it easier to add filters from another location?